src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -112,7 +112,7 @@ public void execute_duplicatePersonFilteredList_failure() {
         showPersonAtIndex(model, INDEX_FIRST_PERSON);
 
         // edit person in filtered list into a duplicate in address book
-        Person personInList = model.getAddressBook().getPersonList().get(INDEX_SECOND_PERSON.getZeroBased());
+        Person personInList = (Person) model.getAddressBook().getPersonList().get(INDEX_SECOND_PERSON.getZeroBased());
```

Comment:
I need it to be casted as I will receive an error where getPersonList().get is returning an object instead of person therefore there will be a need to cast it

Reply:
![image](https://user-images.githubusercontent.com/70756909/160218926-3bba8c0a-11d1-4528-b995-a2c55ce46162.png)
Are you using the right Java version?